src/backend/distributed/commands/function.c

```diff
@@ -192,6 +202,22 @@ create_distributed_function(PG_FUNCTION_ARGS)
 
 	ObjectAddressSet(functionAddress, ProcedureRelationId, funcOid);
 
+	if (IsCreateDistributedFunctionCallIdempotent(functionAddress,
+												  distributionArgumentName,
+												  colocateWithTableNameDefault,
+												  forceDelegationAddress))
+	{
+		char *schemaName = get_namespace_name(get_func_namespace(funcOid));
+		char *functionName = get_func_name(funcOid);
+		char *qualifiedName = quote_qualified_identifier(schemaName, functionName);
+		ereport(NOTICE, (errmsg("procedure %s is already distributed", qualifiedName),
```

Comment:
Should we make `procedure` or `function` based on the actual function? Because in Postgres procedure and functions are slightly different things  and could easily confuse